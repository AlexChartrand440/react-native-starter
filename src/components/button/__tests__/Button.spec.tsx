import 'react-native';
import * as React from 'react';
import * as testRenderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

const Button = () => null;

test('renders correctly', () => {
  const tree = testRenderer.create(<Button />).toJSON();

  expect(tree).toMatchSnapshot();
});

// test('Button renders children text', () => {
//   const text = 'Sample text';
//   const button = shallow(<Button />);
//   expect(button.findWhere(c => c.text() === text)).toBeTruthy();
// });
