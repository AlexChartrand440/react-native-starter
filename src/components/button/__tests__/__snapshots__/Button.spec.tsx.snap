// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType="button"
  accessibilityTraits={
    Array [
      "button",
    ]
  }
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "var(--primary-color)",
          "borderRadius": 4,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "black",
            "fontSize": 18,
            "paddingBottom": 12,
            "paddingLeft": 12,
            "paddingRight": 12,
            "paddingTop": 12,
            "textAlign": "center",
          },
        ]
      }
    >
      test
    </Text>
  </View>
</View>
`;
